build.gradle.kts

```diff
@@ -7,15 +7,15 @@ plugins {
     `java-library`
     jacoco
     id("org.sonarqube") version "3.0"
+    id("com.vanniktech.maven.publish") version "0.13.0"
 }
 
 allprojects {
     version = "1.0.0"
-    group = "de.ppi.deepsampler"
+    group = "de.ppi"
 
     apply(plugin = "java-library")
     java {
-        withSourcesJar()
```

Comment:
Is it really correct to remove `withSourceJar()`? I believe we need this if we want to publish sources to maven central...?

Reply:
Yes, the maven publish plugin will generate it, so its duplicate at this point.